### ISSUE TITLE
metameetup post

### ISSUE THREAD

**Issue by maelle** [rladies/rladies.github.io#348]
I was looking for https://github.com/rladies/rladies.github.io/blob/963574d263e6a3d1b6d3c50ec6484af736fed2d2/content/blog/2020-06-24-spanishmetameetup/index.es.Rmd because of https://github.com/rladies/rladiesguide/pull/150

Is it absent because it is in Spanish?

**Comment by drmowinckels** [comment]
yes, when I integrated the blog into the main website, this was a consequence I had not thought through thoroughly. And I have since come to regret it. 

I have thought of a fix, but it would require quite some work. 

1. place the rladies hugo theme in its own repo
2. reinstate the blog as its own subdomain using the rladies theme

This would allow for blogposts in languages we dont have active on the main site.

**Comment by maelle** [comment]
and the blog could be something easier à la Quarto (thinking of that since that's the case for cynkra.com vs blog.cynkra.com)

**Comment by maelle** [comment]
would news live on the blog too
